_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -5871,15 +5871,15 @@ mfxStatus ConfigureExecuteParams(
 
                         if (!executeParams.dstRects.empty())
                         {
-                            if (executeParams.dstRects.size() < StreamCount)
+                            if (StreamCount > executeParams.initialStreamNum)
```

Comment:
Thank you. Now it looks correct. But this will look better:
```
                        if (executeParams.dstRects.empty())
                        {
                            executeParams.initialStreamNum = StreamCount;
                        }
                        else
                        {
                            if (StreamCount > executeParams.initialStreamNum)
                                return MFX_ERR_INCOMPATIBLE_VIDEO_PARAM;
                        }
                        executeParams.dstRects.resize(StreamCount);
```